data_steward/tools/create_tier.py

```diff
@@ -17,11 +19,18 @@
 
 TIER_LIST = ['controlled', 'registered']
 DEID_STAGE_LIST = ['deid', 'deid_base', 'deid_clean']
+BASE = 'base'
+
 SCOPES = [
     'https://www.googleapis.com/auth/bigquery',
     'https://www.googleapis.com/auth/devstorage.read_write',
 ]
 
+ADD_ETL_METADATA_QUERY = JINJA_ENV.from_string("""
+INSERT INTO `{project_id}.{dataset_id}._cdr_metadata` (qa_handoff_date)
+VALUES ({field_value})
```

Comment:
I think you need to wrap this value in single quotes.
```suggestion
VALUES ('{field_value}')
```